java/android/nnstreamer/src/main/jni/nnstreamer-native-common.c

```diff
@@ -427,6 +441,28 @@ nnstreamer_native_initialize (JNIEnv * env, jobject context)
   return (g_nns_is_initialized == TRUE);
 }
 
+/**
+ * @brief Release NNStreamer, close internal resources.
+ */
+void
+nnstreamer_native_finalize (void)
+{
+  _ml_logi ("Called native finalize.");
+
+  G_LOCK (nns_native_lock);
+
+#if defined(__ANDROID__)
+#if defined(ENABLE_ML_AGENT)
+  ml_agent_finalize ();
+#endif
+#endif
```

Comment:
It would be nice to modify it as follows. What do you think?

```
#if defined(__ANDROID__) && defined(ENABLE_ML_AGENT)
  ml_agent_finalize ();
#endif
```

Reply:
For future work, case if android and not ml-agent.